source/common/stats/thread_local_store.h

```diff
@@ -130,8 +130,7 @@ class TlsScope : public Scope {
 };
 
 /**
- * Store implementation with thread local caching. For design details see
- * https://github.com/envoyproxy/envoy/blob/master/docs/stats.md
```

Comment:
Let's merge this with another PR and update the link rather than remove
https://github.com/envoyproxy/envoy/blob/master/source/docs/stats.md looks like it works?